scripts/plasticc/plasticc.py

```diff
@@ -0,0 +1,201 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datasets
+from datasets import Features, Value, Array2D, Sequence
+from datasets.data_files import DataFilesPatternsDict
+from pathlib import Path
+import itertools
+import h5py
+import numpy as np
+
+_CITATION = """\
+@article{Kessler_2019,
+   title={Models and Simulations for the Photometric LSST Astronomical Time Series Classification Challenge (PLAsTiCC)},
+   volume={131},
+   ISSN={1538-3873},
+   url={http://dx.doi.org/10.1088/1538-3873/ab26f1},
+   DOI={10.1088/1538-3873/ab26f1},
+   number={1003},
+   journal={Publications of the Astronomical Society of the Pacific},
+   publisher={IOP Publishing},
+   author={Kessler, R. and Narayan, G. and Avelino, A. and Bachelet, E. and Biswas, R. and Brown, P. J. and Chernoff, D. F. and Connolly, A. J. and Dai, M. and Daniel, S. and Stefano, R. Di and Drout, M. R. and Galbany, L. and González-Gaitán, S. and Graham, M. L. and Hložek, R. and Ishida, E. E. O. and Guillochon, J. and Jha, S. W. and Jones, D. O. and Mandel, K. S. and Muthukrishna, D. and O’Grady, A. and Peters, C. M. and Pierel, J. R. and Ponder, K. A. and Prša, A. and Rodney, S. and Villar, V. A.},
+   year={2019},
+   month=jul, pages={094501} }
+"""
+
+_DESCRIPTION = """\
+The Photometric LSST Astronomical Time-Series Classification Challenge (PLAsTiCC) is a community-wide challenge to spur development of algorithms to classify astronomical transients. The Large Synoptic Survey Telescope (LSST) will discover tens of thousands of transient phenomena every single night. To deal with this massive onset of data, automated algorithms to classify and sort astronomical transients are crucial.
+
+PLAsTiCC, based on the highly successful Supernova Photometric Classification Challenge, will consist of a set of realistic LSST simulations of a variety of transient and variable phenomena. The challenge will be publicly available on a popular data science platform, encouraging algorithm submissions from outside the Astronomy community.
+"""
+
+_HOMEPAGE = "https://zenodo.org/records/2539456"
+
+_LICENSE = "CC BY 4.0"
+
+_VERSION = "0.0.1"
+
+_FLOAT_FEATURES = [
+        'hostgal_photoz',
+        'hostgal_specz',
+        'redshift',
+    ]
+
+_STR_FEATURES = [
+        'obj_type',
+    ]
+
+_PLASTICC_CLASS_MAPPING = {
+    90: "SNIa",
+    67: "SNIa-91bg",
+    52: "SNIax",
+    42: "SNII",
+    62: "SNIbc",
+    95: "SLSN-I",
+    15: "TDE",
+    64: "KN",
+    88: "AGN",
+    92: "RRL",
+    65: "M-dwarf",
+    16: "EB",
+    53: "Mira",
+    6: "MicroLens-Single",
+    99: "extra",
+    991: "MicroLens-Binary",
+    992: "ILOT",
+    993: "CaRT",
+    994: "PISN",
+    995: "MicroLens-String"
+}
+
+_PLASTICC_BANDS = ['u', 'g', 'r', 'i', 'z', 'Y']
```

Comment:
I think referring to `_BANDS` is more versatile than referring to `_PLASTICC_BANDS`. We can then try to make this a default.